lib/manager/github-actions/index.ts

```diff
@@ -4,7 +4,7 @@ export { extractPackageFile };
 
 export const defaultConfig = {
   fileMatch: [
-    '^(workflow-templates|\\.github\\/workflows)\\/[^/]+\\.ya?ml$',
+    '^(workflow-templates|(.*)\\.github\\/workflows)\\/[^/]+\\.ya?ml$',
```

Comment:
Please use a Conventional Commits [^cc] style PR title:

`feat(github actions): discover workflow files in subfolders`

You're adding a new feature to Renovate, so it's a `feat:` type change, and the scope is `github actions` because you're changing something related to GitHub Actions. 😉 

[^cc]: https://www.conventionalcommits.org/en/v1.0.0/

Reply:
Thanks, all done.